client/src/app/main/dashboard/dashboard-create-ticket/create-ticket-form.js

```diff
@@ -29,6 +29,7 @@ class CreateTicketForm extends React.Component {
     };
 
     state = {
+        display: false,
```

Comment:
shouldn't this be `true` by default? Also maybe change name to `formDisplayed`

Reply:
okey